dags/intacct_historical.py

```diff
@@ -64,21 +65,23 @@
     )
 
     for location, connector_id in list_of_connectors.items():
-
-        fivetran_sync_start = FivetranOperator(
-                task_id='intacct-task-{}'.format(location),
+        with TaskGroup(f'intacct-{location}', prefix_group_id=False):
+            fivetran_sync_start = FivetranOperator(
+                task_id=f'intacct-task-{location}',
                 fivetran_conn_id='fivetran',
-                connector_id='{}'.format(connector_id)
-        )
-
-        fivetran_sync_wait = FivetranSensor(
-                task_id='intacct-sensor-{}'.format(location),
+                connector_id=connector_id,
+            )
+
+            # It's best if the sensor starts before the Fivetran sync is triggered to avoid any
+            # chance of it missing the Fivetran sync happening, so we give it a higher priority and
+            # don't set it as downstream of the sync start operator.
+            fivetran_sync_wait = FivetranSensor(
+                task_id=f'intacct-sensor-{location}',
                 fivetran_conn_id='fivetran',
-                connector_id='{}'.format(connector_id),
+                connector_id=connector_id,
                 poke_interval=5,
-                timeout=3*60*60,  # Timeout of 3 hours
-        )
-
-        fivetran_sync_start >> fivetran_sync_wait >> fivetran_sensors_complete
-
+                execution_timeout=timedelta(hours=3),
+                priority_weight=fivetran_sync_start.priority_weight + 1,
+            )
 
+            fivetran_sync_wait >> fivetran_sensors_complete
```

Comment:
I believe this should stil be:

```python
fivetran_sync_start >> fivetran_sync_wait >> fivetran_sensors_complete
```

Reply:
As originally envisioned, this change is the point of the PR, because if you start the Fivetran sync and only then start the sensor there's no guarantee how quickly Airflow will schedule it, and in some cases the sensor is starting minutes later (Airflow busy with other stuff) after the Fivetran sync has completed.

Reply:
would in this case the sensor just say I don't see an active sync in progress in Fivetran and exit successfully? In this case this would be fine?

Reply:
> would in this case the sensor just say I don't see an active sync in progress in Fivetran and exit successfully? In this case this would be fine?

No (see other thread).